de.dentrassi.pm.aspect.mvnosgi/src/de/dentrassi/pm/aspect/mvnosgi/internal/VirtualizerImpl.java

```diff
@@ -0,0 +1,67 @@
+package de.dentrassi.pm.aspect.mvnosgi.internal;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.osgi.framework.Version;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.dentrassi.pm.aspect.common.osgi.OsgiAspectFactory;
+import de.dentrassi.pm.aspect.virtual.Virtualizer;
+import de.dentrassi.pm.common.ArtifactInformation;
+import de.dentrassi.pm.osgi.bundle.BundleInformation;
+
+public class VirtualizerImpl implements Virtualizer
+{
+    private final static Logger logger = LoggerFactory.getLogger ( VirtualizerImpl.class );
+
+    private final static String GROUP_ID = "groupid";
```

Comment:
The group ID should be configurable.
But this option should only be available if the aspect get's activated.
Could we add configuration to aspects?